extensions/yarn/main.ts

```diff
@@ -93,17 +102,33 @@ if (Deno.args.length >= 1
         console.log(`[${green("phylum")}] Packages built successfully.`);
     }
 } else {
-    let status = await Deno.run({ cmd: ['yarn', ...Deno.args] }).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'yarn',
+        args: [...Deno.args],
+        exceptions: {
+            read: true,
+            write: ['~/.cache/node', '~/.cache/yarn', '~/.yarn', './'],
+            run: false,
+            net: true,
+        }
+    })
```

Comment:
This has a `read` + `net` as true, which is a bit unfortunate for sandboxing. I don't think we have a better solution here but it might be worth it at some point to go through the `yarn` subcommands and see if there's anything else we want to protect against here.

Otherwise this is basically running unsandboxed, which is totally fine and will certainly be required by some extensions.